### ISSUE TITLE
PTR/rDNS Support

### ISSUE THREAD

**Issue by SlothCroissant** [AdguardTeam/AdGuardHome#1562]
NOTE: This is a follow-up from a question I'd had in #1458

I'm looking for PTR/reverse DNS lookup support in AGH. For example - I'd like to do a `dig -x 10.1.1.5` and have it respond with `svr01`. Right now, the PTR call isn't responded to by AGH:

```
user@pc:~$ dig -x 10.1.1.5 @10.1.1.2

; <<>> DiG 9.11.5-P4-5.1ubuntu2.1-Ubuntu <<>> -x 10.1.1.5 @10.1.1.2
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 9634
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;5.1.1.10.in-addr.arpa.		IN	PTR

;; Query time: 24 msec
;; SERVER: 10.1.1.2#53(10.1.1.2)
;; WHEN: Mon Mar 09 19:37:25 CDT 2020
;; MSG SIZE  rcvd: 50
```

This works on PiHole, and I'd love to see it incorporated into AGH. I think it could use the same IP/name sources that AGH already leverages - DNS Rewrites and `/etc/hosts` entries (which i believe is coming in v102?).

I saw this was mildly referenced in #1478, but PTR wasn't landed on there, so making a new request for it specifically for tracking.

**Comment by ameshkov** [comment]
@szolin have we implemented it as a part of #1478?

**Comment by szolin** [comment]
> have we implemented it as a part of #1478?

No, PTR isn't implemented.
In #1478 we just added the code to respond to A/AAAA requests using info from `/etc/hosts`.
